src/configupdater/option.py

```diff
@@ -101,6 +102,27 @@ def _join_multiline_value(self):
             self._value = "\n".join(self._values).rstrip()
             self._multiline_value_joined = True
 
+    def _get_delim(self, determine_suffix=True) -> str:
+        document = self._document()
+        opts = getattr(document, "syntax_options", None) or {}
+        value = self._value
+        space = self._space_around_delimiters or opts.get("space_around_delimiters")
+        if space:
+            if determine_suffix:
+                # no space is needed if we use multi-line arguments
+                suffix = "" if str(value).startswith("\n") else " "
+            else:
+                suffix = " "
+            delim = f" {self._delimiter}{suffix}"
+        else:
+            delim = self._delimiter
+        return delim
+
+    def value_start_idx(self) -> int:
```

Comment:
Thanks but I think this could be helpful also for users which want to indent not by a static number like 4 but always with the width of where the value would start after `option = `.